packages/svelte/src/index-server.js

```diff
@@ -35,6 +36,21 @@ export function unmount() {
 
 export async function tick() {}
 
+/** @type {AbortController | null} */
+let controller = null;
+
+export function getAbortSignal() {
+	if (controller === null) {
+		const c = (controller = new AbortController());
+		queueMicrotask(() => {
+			c.abort(STALE_REACTION);
+			controller = null;
+		});
+	}
+
+	return controller.signal;
+}
```

Comment:
I think this needs to change -- ideally this abort signal is aborted at the end of `render` rather than after a microtask, as a microtask will both be "too late" and also will interfere with future async SSR work